File: 1833-find-the-highest-altitude/solution.ts
function largestAltitude(gain: number[]): number {
  let now = 0;
  let answer = 0;
  for (let i = 0; i < gain.length; i++) {
    now += gain[i];
    answer = answer > now ? answer : now;
  }
  return answer;
}

File: 1833-find-the-highest-altitude/solution.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int maxG = 0, now = 0;
        for(auto &g : gain) {
            now += g;
            maxG = max(maxG, now);
        }
        return maxG;
    }
};
